src/mol-util/marker-action.ts

```diff
@@ -35,19 +37,36 @@ export namespace MarkerActions {
         MarkerAction.Select | MarkerAction.Deselect | MarkerAction.Toggle |
         MarkerAction.Clear
     ) as MarkerActions;
+    export const Focusing = (
+        MarkerAction.Focus | MarkerAction.RemoveFocus |
+        MarkerAction.Clear
+    ) as MarkerActions;
 
     export function isReverse(a: MarkerAction, b: MarkerAction) {
         return (
             (a === MarkerAction.Highlight && b === MarkerAction.RemoveHighlight) ||
             (a === MarkerAction.RemoveHighlight && b === MarkerAction.Highlight) ||
             (a === MarkerAction.Select && b === MarkerAction.Deselect) ||
             (a === MarkerAction.Deselect && b === MarkerAction.Select) ||
-            (a === MarkerAction.Toggle && b === MarkerAction.Toggle)
+            (a === MarkerAction.Toggle && b === MarkerAction.Toggle) ||
+            (a === MarkerAction.Focus && b === MarkerAction.RemoveFocus) ||
+            (a === MarkerAction.RemoveFocus && b === MarkerAction.Focus)
         );
     }
 }
 
-export function setMarkerValue(array: Uint8Array, status: 0 | 1 | 2 | 3, count: number) {
+/** Bitflags for 3 types of marker (1 = highlighted, 2 = selected, 4 = focused) */
+export type MarkerValue = 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7;
+export const MarkerValue = {
+    Highlighted: 1,
+    Selected: 2,
+    Focused: 4,
```

Comment:
@arose do you think we could find a use case for the focused stated in 3D visualization as well?
Otherwise I've suggested to Adam to implement this login only for the Sequence component.

Reply:
i found a way to treat focus markers separate, but in an efficient way